docsrc/index.adoc.tmpl

```diff
@@ -1,12 +1,19 @@
 {{ $static := datasource "static" }}
 {{ $rules := datasource "rules" }}
+////
+{{ $static.generated_content_warning }}
+////
 
 = HACBS Enterprise Contract Policies
 
+:toc: left
+:icons: font
+:numbered:
+
 == About
 
 {{ range $static.about }}
-+++<p>{{ . }}</p>+++
++++{{ . }}+++
```

Comment:
We probably don't need the pasthrough block any more here since `about` has no HTML in it
```suggestion
{{ . }}
```

Reply:
There's one link to rego, so that's the reason for it. (The problem goes away if we stop having different formats...)